templates/kubernetes/terraform/modules/kubernetes/cert_manager.tf

```diff
@@ -2,6 +2,68 @@ locals {
   cert_manager_version     = "1.1.0"
   cluster_issuer_name      = var.cert_manager_use_production_acme_environment ? "clusterissuer-letsencrypt-production" : "clusterissuer-letsencrypt-staging"
   cert_manager_acme_server = var.cert_manager_use_production_acme_environment ? "https://acme-v02.api.letsencrypt.org/directory" : "https://acme-staging-v02.api.letsencrypt.org/directory"
+
```

Comment:
Switched this to an object which is yaml encoded rather than templating a file because it's a bit more dynamic now. Removed the `ClusterIssuer` yaml and also the cert-manager CRD yaml which is not used anymore.